hal/src/main/native/athena/HAL.cpp

```diff
@@ -594,6 +594,14 @@ HAL_Bool HAL_Initialize(int32_t timeout, int32_t mode) {
   });
 
   if (!SetupNowRio()) {
+    fmt::print(stderr,
+               "Failed to run SetupNowRio(). This is a fatal error. The "
+               "process is being terminated.\n");
+    std::fflush(stderr);
```

Comment:
Do we set stderr to buffered somewhere? It's unbuffered by default.

Reply:
Doubt it, but it also doesn't hurt to do it...